themes/skycoin/layouts/shortcodes/subhead.html

```diff
@@ -8,6 +8,7 @@
   <a class="nav__item" href="{{ .Site.Params.urlGit }}/issues/new?body=Source%20File%3A%20%0A%0A{{ .Page.Params.filename }}">Report issue</a>
   <a class="nav__item" href="https://t.me/Skycoin">Discuss</a>
 </div>
+<div class="space"></div>
```

Comment:
No need to add an element for this . Better add CSS rules in the corresponding theme sass files . I'll accept the PR anyways for the moment .